frontend/src/views/queries/components/QuerySquare.js

```diff
@@ -1,15 +1,43 @@
-import React from 'react'
-import { Input } from 'antd'
+import React, { useEffect } from 'react';
+import keywords from './SintaxDictionary'
 
 const QuerySquare = props => {
-  const { TextArea } = Input
+
+  useEffect(() => {
```

Comment:
why not used a input ? the input has several events that you could us instead of that.. You could customize the input size and use the events available for the input

Reply:
Input is not used because it does not allow the text to be edited as required to recognize the syntax and mark the reserved words of the language